Topic1_Audio_Player/src/PlayState.java

```diff
@@ -0,0 +1,19 @@
+
+public class playState implements State{
+	String current = "play";
```

Comment:
Warning: The different states like play, stop, etc, should be encapsulated on a Enum.